docs/kyma/docs/04-02-local-installation-from-release.md

```diff
@@ -142,13 +143,13 @@ http://192.168.64.44:30000
 
 ## Enable Horizontal Pod Autoscaler (HPA)
 
-By default, the Horizontal Pod Autoscaler is not enabled in your local Kyma installation, so you need to enable it manually.
+By default, the Horizontal Pod Autoscaler (HPA) is not enabled in your local Kyma installation, so you need to enable it manually.
 
-Kyma uses the autoscaling/v1 stable version, which only provides support for CPU autoscaling. Once enabled, HPA automatically scales the number of lambda function Pods based on observed CPU utilization.
+Kyma uses the `autoscaling/v1` stable version, which supports only CPU autoscaling. Once enabled, HPA automatically scales the number of lambda function Pods based on the observed CPU utilization.
 
->**NOTE:** The autoscaling/v1 version does not support custom metrics. To use such metrics, you need the autoscaling/v2beta2 version.
+>**NOTE:** The `autoscaling/v1` version does not support custom metrics. To use such metrics, you need the `autoscaling/v2beta2` version.
```

Comment:
@Demonsthere shouldn't be v2beta1?

Reply:
Hi, this is kubernetes version related. `v2beta2` is available only from `v1.12` https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#horizontalpodautoscaler-v1-autoscaling